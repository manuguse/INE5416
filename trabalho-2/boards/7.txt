3 7 5 3 - - - -
- - - - - - - -
- 1 - 2 - - - -
- - - - - 1 - -
- - 6 4 - - - -
- - 3 - 3 - 5 -
- - - - - - - -
- 5 - - - - 4 -