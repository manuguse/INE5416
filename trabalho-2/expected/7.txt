3 7 5 3 6 2 1 2
1 2 4 1 2 1 2 1
3 1 3 2 1 4 3 2
2 5 7 1 3 1 2 1
4 1 6 4 2 4 1 3
3 2 3 2 3 1 5 1
2 1 2 1 2 3 1 2
1 5 1 2 1 2 4 1